compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -14117,6 +14117,168 @@ bool isSettingOp(uint8_t byteValue, TR::InstOpCode::Mnemonic SI_opcode)
    }
 
 
+
```

Comment:
Let's clean up this area of code and replace the triple new line with a single new line.